### PR TITLE
Add revision and version info to bug-report

### PR DESCRIPTION
**Pull request by ostromart**
The following Istio control plane revisions/versions were found in the cluster:
Revision istio-169:
&version.MeshInfo{
    {
        Component: "istiod",
        Info:      version.BuildInfo{Version:"1.6.9-gke.0-96a48092d219e3e087343165180f60b396c4d421", GitRevision:"dirty", GolangVersion:"", BuildStatus:"Modified", GitTag:"1.6.9-gke.0-96a48092d219e3e087343165180f60b396c4d421"},
    },
}

The following proxy revisions/versions were found in the cluster:
Revision istio-169: Versions {1.6.9-gke.0}

### CODE REVIEW

**Review by stewartbutler — APPROVED** [review]

File: tools/bug-report/pkg/bugreport/bugreport.go [hunk 1]
```diff
@@ -174,6 +179,67 @@ func runBugReportCommand(_ *cobra.Command, logOpts *log.Options) error {
...
+	}
+	return out
+}
+
+// getIstioVersions returns a mapping of revision to aggregated version string for Istio components and revision to
+// slice of versions for proxies. Any errors are embedded in the revision strings.
+func getIstioVersions(kubeconfig, configContext, istioNamespace string, revisions []string) (map[string]string, map[string][]string) {
+	istioVersions := make(map[string]string)
+	proxyVersions := make(map[string][]string)
+	for _, revision := range revisions {
```
Comment (howardjohn) [inline tools/bug-report/pkg/bugreport/bugreport.go:216]: won't this list end up with possible many duplicate labels, or (MANY labels, if it includes sidecars, which I think it does)? Should revisions be deduped?

Comment (ostromart) [reply tools/bug-report/pkg/bugreport/bugreport.go:217]: good catch, done.

Comment (howardjohn) [reply tools/bug-report/pkg/bugreport/bugreport.go:217]: you are still calling `getIstioVersion` and `GetProxyInfo` which is expensive for each one I think.

Comment (ostromart) [reply tools/bug-report/pkg/bugreport/bugreport.go:217]: yes, but they're called just once per revision - i don't see a way to call them fewer times than that.

**Review by howardjohn — APPROVED** [review]

**Pull request merged**